src/open_inwoner/plans/views.py

```diff
@@ -278,7 +286,7 @@ class PlanCreateView(
     BaseBreadcrumbMixin,
     CreateView,
 ):
-    template_name = "pages/plans/create.html"
+    template_name = "pages/plans/create-no-template.html"
```

Comment:
We don't have the `-no-template` for the view or url names, why here? Generally i think it would be nice to have the naming of the templates go along with the naming of the urls, so `collaborate:plan_create` <-> `pages/plans/plan-create.html`. Probably outside the scope of this PR though.